integration-tests/jest/1-run-notebook.integration-test.js

```diff
@@ -2,4 +2,4 @@ const { registerTest } = require('./jest-utils')
 const { testRunNotebook } = require('../tests/run-notebook')
 
 
-registerTest(testRunNotebook)
+registerTest(testRunNotebook, 1)
```

Comment:
This has several failure modes that I've seen frequently. The most common is the "403" response. The next most common is probably where the notebook is open but hasn't evaluated the expression for some reason. (It happened just this morning on alpha: https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/5492/workflows/b550addd-1e83-4483-8a08-797e588315da/jobs/28464). I really feel like we should be able to fix these up.